README.md

```diff
@@ -7,5 +7,5 @@ UX elements for Jellyfin, including logos, fonts, and other elements shared acro
 
 ---
 <p align="center">
-Please see the <a href="https://docs.jellyfin.org/general/contributing/branding-guide.html">branding guide</a> for more information on using these assets.
+Please see the <a href="https://jellyfin.org/docs/general/contributing/branding.html">branding guide</a> for more information on using these assets.
```

Comment:
```suggestion
Please see the <a href="https://docs.jellyfin.org/general/contributing/branding.html">branding guide</a> for more information on using these assets.
```

Reply:
We are keeping the subdomain just in case we ever migrate the documentation to a serparate host.